proto/encoded.go.template

```diff
@@ -70,7 +70,7 @@ func (this *EncodedThing) VerboseEqual(that interface{}) error {
 	if thatP, ok := that.(EncodedThing); ok {
 		return this.Thing.VerboseEqual(&thatP.Thing)
 	}
-	return fmt.Errorf("types don't match: %T != EncodedThing")
+	return fmt.Errorf("types don't match: %T != EncodedThing", that.(*EncodedThing))
```

Comment:
Won't these type assertions always panic if executed? Based on the `if`-s before we know that `that` cannot possible be interpreted as an `*EncodedThing`. Probably you meant to pass `that` itself as the second argument (without any coercion).